src/workerd/io/actor-cache.h

```diff
@@ -599,6 +638,18 @@ class ActorCache final: public ActorCacheInterface {
     auto begin() { return inner.begin(); }
     auto end() { return inner.end(); }
 
+    // The way we coalesce reads means we update `Entry`s while they're still in the `dirtyList`.
+    // This means the size of the list may change, without us adding or removing an Entry.
+    // To ensure proper accounting, we have to modify the size when we modify an Entry.
+    void decreaseSize(size_t bytes) {
+      innerSize -= bytes;
```

Comment:
Some kind of assertion that `innerSize >= bytes` seems important here if we need to keep this.